modules/local/wipertools/gather.nf

```diff
@@ -0,0 +1,43 @@
+process GATHER {
```

Comment:
Is it possible to name it in nf-core standards such as `WIPERTOOLS_GATHER`?

Reply:
I agree - I think as it's already a standalone package and once it goes onto bioconda it would be perfect for an official module even, so renaming to the standard system will make it easier for you to convert this in the future @mazzalab :)